docs/tutorials/articles/colab_with_ngrok/index.md

```diff
@@ -115,14 +115,21 @@ To address this issue, you can modify the *change_delay* parameter in one of the
 
 - **Locally**: You can make changes to this parameter locally, directly within the Taipy visual
   element.
-  ```python
-  <|{text}|input|change_delay=800|>
-  ```
+
+=== "Markdown" 
+   ```python
+   <|{text}|input|change_delay=800|>
+   ```
+=== "Python"
+   ```python
+   tgb.input("{text}", change_delay=800)
+   ```
 
 - **Globally**: To adjust the delay for all of Taipy's visual elements.
-  ```python
-  Gui(pages=pages).run(change_delay=800, ngrok_token=<Authtoken>)
-  ```
+
+```python
+Gui(pages=pages).run(change_delay=800, ngrok_token=<Authtoken>)
+```
```

Comment:
must be indended (4 spaces) if it is part of the bullet point.

Reply:
Same thing. I didn't know this was possible